code-studio/src/js/components/progress/stage_progress.jsx

```diff
@@ -46,4 +44,6 @@ const StageProgress = React.createClass({
     );
   }
 });
-module.exports = StageProgress;
+export default connect((state, ownProps) => ({
+  levels: ownProps.levels || state.stages[0].levels
```

Comment:
If it's not passed in via `props` it's because we're rendering `StageProgress` directly (in the header) and we're only showing one stage.